internal_review/quality_of_life/borough/traffic_injuries_fatalities.csv

```diff
@@ -1,4 +1,4 @@
-borough,safety_1014_trafficinjuries,safety_1014_trafficinjuries_ped,safety_1014_trafficinjuries_cycle,safety_1014_trafficinjuries_motorist,safety_1014_trafficfatalities,safety_1620_trafficinjuries,safety_1620_trafficinjuries_ped,safety_1620_trafficinjuries_cycle,safety_1620_trafficinjuries_motorist,safety_1620_trafficfatalities
+borough,safety_1014_trafficinjuries_count,safety_1014_trafficinjuries_ped_count,safety_1014_trafficinjuries_cycle_count,safety_1014_trafficinjuries_motorist_count,safety_1014_trafficfatalities_count,safety_1620_trafficinjuries_count,safety_1620_trafficinjuries_ped_count,safety_1620_trafficinjuries_cycle_count,safety_1620_trafficinjuries_motorist_count,safety_1620_trafficfatalities_count
```

Comment:
I think these should be labeled as rate since the measure is annual average per 100 street miles